service/src/main/java/com/epam/esm/CertificateService.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm;
+
+import com.epam.esm.dto.CertificateDTO;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+public interface CertificateService {
+    List<CertificateDTO> getAllCertificates();
+
+    void saveCertificate(CertificateDTO certificateDTO);
+
+    CertificateDTO getCertificate(int id);
+
+    void deleteCertificate(int id);
+
+    @Transactional
```

Comment:
Why did you place here this annotation? Why you didn't place it also for other methods?